server/app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,2 @@
-module ApplicationCable
-  class Connection < ActionCable::Connection::Base
-  end
+class ApplicationCable::Connection < ActionCable::Connection::Base
```

Comment:
Do we prefer this syntax or do we like the nested syntax better?

I don't care either way, but I've setup the rubocop rule to fix it to this compact syntax.

Reply:
I think I have a slight preference to use the nested syntax but that's just because I'm more used to it. I don't have a strong preference regardless

Reply:
I'll change it to that. I think that's what I'm more used to too.